src/main/kotlin/com/sourcegraph/cody/chat/AgentChatSession.kt

```diff
@@ -115,21 +115,15 @@ private constructor(
                   addEnhancedContext = chatPanel.isEnhancedContextEnabled(),
                   contextFiles = contextItems)
 
-          try {
-            val request =
-                agent.server.chatSubmitMessage(
-                    ChatSubmitMessageParams(connectionId.get().get(), message))
-
-            GraphQlLogger.logCodyEvent(project, "chat-question", "submitted")
-
-            createCancellationToken(
-                onCancel = { request.cancel(true) },
-                onFinish = { GraphQlLogger.logCodyEvent(project, "chat-question", "executed") })
-          } catch (e: Exception) {
-            createCancellationToken(onCancel = {}, onFinish = {})
```

Comment:
Why that part was removed?
I think removal of the `createCancellationToken(onCancel = {}, onFinish = {})` will cause "Generating" button to be stuck forever. also it removes error handling for all errors, and some of them we want to display in the chat.

Please do an experiment disable network connection, then type a chat message. What happens with, and without your changes?

Reply:
I have removed this try-catch block because it seems to duplicate the logic inside the `onFailure` callback just a few lines below. 

I have added a second test plan. Behaviour is the same as in main.